lib/ansible/module_utils/iosxr.py

```diff
@@ -106,6 +106,11 @@ def load_config(module, commands, commit=False, replace=False, comment=None):
         module.fail_json(msg=err, commands=commands, rc=rc)
 
     rc, diff, err = exec_command(module, 'show commit changes diff')
+    if rc != 0:
```

Comment:
This assumes the rc is nonzero because of an unsupported command. Any output we can depend upon, because other things can make a command rc nonzero